charts/spire/charts/spire-server/README.md

```diff
@@ -123,6 +123,9 @@ In order to run Tornjak with simple HTTP Connection only, make sure you don't cr
 | controllerManager.service.port | int | `443` |  |
 | controllerManager.service.type | string | `"ClusterIP"` |  |
 | controllerManager.validatingWebhookConfiguration.failurePolicy | string | `"Fail"` |  |
+| dataStore.dbSecret | object | `{"key":"DBPW","name":""}` | The secret should reside in the same namespace as the spire server |
+| dataStore.dbSecret.key | string | `"DBPW"` | The key of the secret object whose value is the dataStore password |
+| dataStore.dbSecret.name | string | `""` | The name of the secret object |
```

Comment:
What do you think of putting this under `datastore.sql.externalSecret` or something like that?  Would an enable flag help here so you don't have to check name is not empty to enable the feature?